lib/api/units/validateHeaders.js

```diff
@@ -0,0 +1,68 @@
+const { HeadersJsonExample } = require('../../validators/headers-json-example');
+
+const APIARY_JSON_HEADER_TYPE = 'application/vnd.apiary.http-headers+json';
+
+function normalizeHeaders(headers) {
```

Comment:
**Heads up:** One of the things we should do in the future is to support headers of the same name (e.g. [Set-Cookie](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie)). I don't think we should do it now, but just keep it in mind that one day in distant future, we might want change the input format for headers to `array` of `{ name, value }` instead of an `object` where names are keys.

Reply:
Also, there is this neat thingy https://github.com/request/caseless

Reply:
Thanks for insights. Didn't find a corresponding issue, created: https://github.com/apiaryio/gavel.js/issues/160